docs/usage/esql.asciidoc

```diff
@@ -0,0 +1,34 @@
+[[esql]]
+=== ES|QL in the Java client
+
+This page enables you to understand and use {ref}/esql.html[ES|QL] in the
+Java client.
+
+
+[discrete]
+[[esql-how-to]]
+==== How to use the ES|QL API
+
+[TO DO]
```

Comment:
Something like:

> The {es-docs}/esql-query-api.html[ES|QL query API] allows you to choose how results should be returned. This includes the {es-docs}/esql-rest.html#esql-rest-format[response format] such as CSV, text or JSON, which can be further fine-tuned with parameters such as column separators or locale.
>
> Since the response format varies widely depending on the chosen format, the {java-client) as a `BinaryData` object, which you can then use according to the format chosen in the request.
>
> The example below shows how to get ES|QL results as CSV and parse them:
>
> [[ Code example to be written ]]